nixos/modules/tasks/filesystems/zfs.nix

```diff
@@ -575,7 +575,7 @@ in
                       * )
                         ${cfgZfs.package}/sbin/zfs load-key "$ds"
                         ;;
-                    esac) < /dev/null # To protect while read ds kl in case anything reads stdin
+                    esac < /dev/null # To protect while read ds kl in case anything reads stdin
```

Comment:
this comment makes us think the grouping was intentional, maybe replacing it with `{ ... }` would be better? should be fine either way though as is now.

Reply:
Good idea -- I've wrapped the block in curly braces instead of parens.